modules/accounting/model/accounting.class.php

```diff
@@ -292,17 +292,18 @@ public function get_vataccount_object($args) {
         #print "$query_vat<br>\n";
 
         $vat = $_lib['storage']->get_row(array('query' => $query_vat));
-        #Det trenger ikke v?re konto oppsett p? 30, 32, 60 og 62
-        #Merk atVatID null bør sjekkes for opprinnelsen, dette bare demper feilmeldingene.
+        #It desn't need our accountplan setup on vat_id 30, 32, 60 and 62
```

Comment:
It desn't need our accountplan setup on vat_id 30, 32, 60 and 62
 desn't => doesn't